applications/nrf_desktop/src/modules/Kconfig.dvfs

```
@@ -41,24 +40,43 @@ config DESKTOP_DVFS_RETRY_COUNT
 	  Number of retries of DVFS frequency change after which DVFS module will report
 	  MODULE_STATE_ERROR.
 
-config DESKTOP_DVFS_SMP_TRANSFER_TIMEOUT_MS
-	int "SMP transfer timeout"
-	default 5000
-	range 1 65535
-	help
-	  Timeout in milliseconds specifying time after which DVFS module will go back to
-	  low frequency after receiving SMP transfer event.
-
-config DESKTOP_DVFS_CONFIG_CHANNEL_TIMEOUT_MS
-	int "Config channel timeout"
-	default 5000
-	range 1 65535
-	help
-	  Timeout in milliseconds specifying time after which DVFS module will go back to
-	  low frequency after reciving config event.
-
 module = DESKTOP_DVFS
 module-str = DVFS
 source "subsys/logging/Kconfig.template.log_config"
 
+dvfs_dependencies = y
+dvfs_state = INITIALIZING
+dfvs_frequency = FREQ_HIGH
+dvfs_timeout = 0
+rsource "Kconfig.dvfs_state_template"
+
+config DESKTOP_DVFS_STATE_LLPM_CONNECTED_DEPENDENCIES
+	bool
+	depends on CAF_BLE_COMMON_EVENTS && (BT_MAX_CONN = 1)
+	default y
+
+dvfs_dependencies = DESKTOP_DVFS_STATE_LLPM_CONNECTED_DEPENDENCIES
+dvfs_state = LLPM_CONNECTED
+dfvs_frequency = FREQ_MEDLOW
+dvfs_timeout = 0
+rsource "Kconfig.dvfs_state_template"
```

We may consider to remove the `dvfs_dependencies` variable and include the dependencies in the following way:

```suggestion
if CAF_BLE_COMMON_EVENTS && (BT_MAX_CONN = 1)

dvfs_state = LLPM_CONNECTED
dfvs_frequency = FREQ_MEDLOW
dvfs_timeout = 0
rsource "Kconfig.dvfs_state_template"

endif
```